packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -196,15 +197,33 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     if (this.state.isLegendHovered || this.state.isLegendSelected) {
       shouldHighlight = this.state.selectedLegendTitle === lineLegendText;
     }
-    const line = (
+    const lineBorderWidth = this.props.lineOptions?.lineBorderWidth
+      ? Number.parseFloat(this.props.lineOptions!.lineBorderWidth!.toString())
+      : 0;
+
+    if (lineBorderWidth > 0) {
+      line.push(
+        <path
+          opacity={shouldHighlight ? 1 : 0.4}
+          d={linePath(lineData)!}
+          fill="transparent"
+          strokeLinecap="square"
+          strokeWidth={3 + lineBorderWidth * 2}
+          stroke={theme!.palette.white}
+        />,
+      );
+    }
+    line.push(
       <path
         opacity={shouldHighlight ? 1 : 0.4}
         d={linePath(lineData)!}
-        fill={'none'}
+        fill="transparent"
```

Comment:
what does this change do.

Reply:
This is same as none , i made it transparent similar to line chart to avoid any confusion.